### PR TITLE
fix(web-app): filter search popover mobile scroll styles

### LINKED ISSUE #462: Popover needs set max height
### Browser

_No response_

### Description

Bug found by Matt - documenting here:
> @andreancardona @francinelucca ...we need to set a height on the popover content because when its parent is position sticky, and when the popover content is tall, you can't get to the bottom of the popover content.
> 
> We could achieve this by setting overflow scroll (whatever is needed for smooth scroll iOS in CSS) and then a max-height that's 100vh minus whatever's above it.
> 
> It'd be nice to not need a new dependency for this, but this library is popular to get around cross browser mobile quirks.
> 
> https://www.npmjs.com/package/react-div-100vh

### Steps to reproduce

- [ ] change viewport size to mobile. In short view you cannot scroll to see the full content in the popover

https://user-images.githubusercontent.com/32720851/172722163-b0e486d0-3341-4def-9ff8-98d9d6224984.mov

### PR DESCRIPTION
**Pull request by alisonjoseph**
closes #462

#### Testing / reviewing

Test filter scroll at mobile view (will need to make height short enough to require a scroll)

http://localhost:3000/assets/components

### CODE REVIEW

File: services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss [hunk 1]
```diff
@@ -102,9 +102,16 @@
 .content {
   width: calc(100vw - var(--scrollbar-width));
   max-width: none;
+  // height of sticky header + search = 96px
+  height: calc(100vh - convert.rem(96px));
```
Comment (francinelucca) [inline services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss:106]: the last section get a little bit cut off for me when scrolled all the way, maybe we should reduce the height a smaller amount? 
image

**Review by francinelucca — CHANGES_REQUESTED** [review]

File: services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss (hunk 1 above)
Comment (alisonjoseph) [reply services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss:106]: Ah ok, I think the issue is the extra space above the search when you open the filter. Looking into this.

**Review by francinelucca — APPROVED** [review]
🚀

**Comment by mattrosno** [comment]
I removed this PR from the sprint, because its associated issue belongs to Sprint 23.

**Comment by andreancardona** [comment]
Going to open an issue for this - but scroll looks good!

Screen Shot 2022-08-29 at 4 01 18 PM

**Review by andreancardona — APPROVED** [review]

**Pull request merged**